src/app/components/crypto-assets/stacks/fungible-token-asset/stacks-fungible-token-asset-item.tsx

```diff
@@ -25,7 +25,7 @@ export const StacksFungibleTokenAssetItem = forwardRefWithAs(
     const avatar = `${formatContractId(contractAddress, contractName)}::${contractAssetName}`;
     const dataTestId = CryptoAssetSelectors.CryptoAssetListItem.replace(
       '{symbol}',
-      symbol.toLowerCase()
+      symbol?.toLowerCase()
```

Comment:
Why did we add this optional chaining value? Adding these anywhere we need is a sign of greater problems in our code.

https://nwcalvank.dev/javascripts-optional-chaining-is-a-code-smell/